packages/appium/docs/en/quickstart/test-rb.md

```diff
@@ -9,9 +9,10 @@ title: Write a Test (Ruby)
 The [AppiumLib](https://github.com/appium/ruby_lib) and the [AppiumLibCore](https://github.com/appium/ruby_lib_core) (**recommended**) are official Appium client libraries in Ruby, which are available via gem under the [appium_lib](https://rubygems.org/gems/appium_lib) and the [appium_lib_core](https://rubygems.org/gems/appium_lib_core) package names. The appium_lib_core inherits from the Selenium Ruby Binding, and the appium_lib inherits from the appium_lib_core, so installing these libraries include the selenium binding. We recommend `appium_lib_core` if you need a less complex client-side solution. The `appium_lib` has some useful methods the core does not have, but for the cost of greater complexity.
 
 ```bash
-gem install appium_lib
+bundle init
+bundle add appium_lib
```

Comment:
adding this via bundler, otherwise there is nothing you can do with `bundle exec` later on